cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,88 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+import { confirmationMessage } from '../plugins/alertMessages';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
+  let user;
+  let title;
```

Comment:
Variables 'user', 'title', 'description', 'content', 'tags', 'nameNextUser', 'emailNextUser', 'passwordNextUser', 'followButtonSelector', 'followButtonText', 'unfollowButtonSelector', 'unfollowButtonText' should be declared inside the scope they are used. Global variables can cause unexpected behavior in tests.

Reply:
'user', 'title', 'description', 'content', 'tags', 'nameNextUser', 'emailNextUser', 'passwordNextUser' - done
'followButtonSelector', 'followButtonText', 'unfollowButtonSelector', 'unfollowButtonText'  - real global variables because we use them at beforeEach and all tests